### PR TITLE
Mhigbyflowers/rebrand touchups

### PR DESCRIPTION
**Pull request by mhigbyflowers**
Aesthetic changes + bug fixes

### Home Page
image

### EHR (conditions, procedures, measurements) 
image

### Surveys
image

### staging.researchallofus.org
Databrowser should transition close to as seemless as possible to staging researchallofus.org
image

image

### CODE REVIEW

File: public-ui/src/app/views/breadcrumb/breadcrumb.component.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <div class="breadcrumb-container">
   <div class="breadcrumb">
-    <a href="https://www.staging.researchallofus.org/">Home</a>
+    <a href="{{allOfUs}}/">Home</a>
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/breadcrumb/breadcrumb.component.html:3]: different variable name may be ?

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/breadcrumb/breadcrumb.component.html:3]: or can be allofusurl or whatever.

**Review by SrushtiGangireddy — APPROVED** [review]

**Comment by calbach** [comment]
This didn't compile so we can't push it - the past 3 circle runs all failed @mhigbyflowers - please fix

**Comment by calbach** [comment]
```
./project.rb build --environment test
...
ERROR in src/app/views/db-header/db-header.component.ts(15,28): error TS2339: Property 'researchAllOfUsUrl' does not exist on type '{ displayTag: string; debug: boolean; clientId: string; tcellappid: string; tcellapikey: string; ...'.
src/app/views/breadcrumb/breadcrumb.component.ts(23,25): error TS2339: Property 'researchAllOfUsUrl' does not exist on type '{ displayTag: string; debug: boolean; clientId: string; tcellappid: string; tcellapikey: string; ...'.
```

**Comment by calbach** [comment]
Looks like you just need to add that property to this file as well: https://github.com/all-of-us/data-browser/blob/62b49e0aa18dcc3ae46b5f2cb562a7d12cacdc5a/public-ui/src/environments/test-env-base.ts

**Pull request merged**